### PR TITLE
Add the disable wanted level

### PR DESCRIPTION
**Pull request by Prophidys**
Add capabilities to disable the wanted level from config file

## Proposed Changes

    - Add a thread to disable wanted level from the config file

### CODE REVIEW

**Comment by izio38** [comment]
Thanks you for the contribution.

We would like to know if you explored other way of doing this ? Maybe with `void SET_MAX_WANTED_LEVEL(int maxWantedLevel);` ? This would avoid another coroutine running on each tick.

Also, please, in order to make a tick function, please use the `ESX.SetInterval`. If there is no other way of doing this, please try to add a longer detection delay (20ms or so).

File: boot/client/main.lua [hunk 1]
```diff
@@ -34,3 +34,21 @@ if Config.EnableHud then
...
+    if GetPlayerWantedLevel(playerId) ~= 0 then
+      SetPlayerWantedLevel(playerId, 0, false)
+      SetPlayerWantedLevelNow(playerId, false)
+    end
+
+    Citizen.Wait(0)
+
+  end)
+
+end
```
Comment (Dislaik) [inline boot/client/main.lua:54]:
``` suggestion
if Config.DisableWantedLevel then

  ESX.SetInterval(20, function()

    local playerId = PlayerId()

    if GetPlayerWantedLevel(playerId) ~= 0 then
      SetPlayerWantedLevel(playerId, 0, false)
      SetPlayerWantedLevelNow(playerId, false)
    end
    
  end)

end
```

**Comment by ArkSeyonet** [comment]
@Dislaik You can remove the Citizen.Wait(0) inside of ESX.SetInterval the 20 in ESX.SetInterval(20, function() is the wait already, so that's a redundant wait.

**Comment by Darkkaiserl** [comment]
@izio38 ,You right, with this it's enough
``` lua
if Config.DisableWantedLevel then
  SetMaxWantedLevel(0)
end
```

**Review by ablax — APPROVED** [review]
LGTM

**Comment by ArkSeyonet** [comment]
Merged PR with current PR about to be pushed for simplicity. Thanks for the contribution. Credit will be added.

**Pull request closed without merging**